Source/XRInteraction/Source/Runtime/Properties/GrabbableProperty.cs

```diff
@@ -187,15 +186,15 @@ private IEnumerator UngrabGrabAndRelease()
 
             yield return new WaitUntil(() => Interactable.isHovered == false && Interactable.isSelected == false);
 
-            if (Interactable.interactorsSelecting.Count > 0)
-            {
-                DirectInteractor directInteractor = (DirectInteractor)Interactable.interactorsSelecting[0];
-                directInteractor.AttemptGrab();
+            //if (Interactable.interactorsSelecting.Count > 0)
```

Comment:
Wy did you uncoment this?

Reply:
DirectInteractor is obsolete and has been deleted, it didn't let me compile. I replaced the code with something that should work.